addons/survey/static/src/scss/survey_form.scss

```diff
@@ -0,0 +1,251 @@
+.o_survey_header {
+    height: 50px;
+}
+.o_survey_main {
+    min-height: calc(100% - 105px);
+    padding-top: 16px;
+}
+
+.o_survey_question_text_box,
```

Comment:
Everthing should be nested under a main class, like o_survey_page_fill or something like that, to know in which context they apply.